dts/bindings/spi/st,stm32-spi-common.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2021 Fabio Baltieri
```

Comment:
This is moved from another source file and should therefore retain the original copyright notice.

Reply:
Fair point, changed.